README.md

```diff
@@ -166,6 +166,14 @@ custom:
               Resource:
                 - "arn:aws:dynamodb:{REGION}:{ACCOUNT_ID}:myTable"
                 - "arn:aws:dynamodb:{REGION}:{ACCOUNT_ID}:myTable/*"
+          # Versioned DataSource configuration
+          versioned: false # (default, not required)
+          # When you enable versioning on a DynamoDB data source, you specify the following fields
+          # read more at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appsync-datasource-deltasyncconfig.html
+          # deltaSyncConfig:
+          #   baseTableTTL: 0 # (defalut, not required) # The amount of time (in minutes) items should be kept in the base table when deleted. Set to 0 to delete items in the base table immediately
+          #   deltaSyncTableName: { Ref: MyTableDelta } # required # The Delta Sync table name
+          #   deltaSyncTableTTL: 60 # (defalut, not required) # The amount of time (in minutes) the delta sync table will keep track of changes
```

Comment:
@yyoshiki41 where do the default come from ? 
what are the min and max values ? 
please fix typo too :) `defalut` = `default`

Reply:
@bboure 

### TTL limitations
Max limitation are not defined.
Min limitations are below.
0 ≦ baseTableTTL
60 ≦ deltaSyncTableTTL

> where do the default come from ?

I searched aws docs but default values is not mentioned in official.
So I adopted min ttl values as default.

---

fixed typo 🙇 
https://github.com/sid88in/serverless-appsync-plugin/pull/313/commits/6e4c551d1c24df01cf3b16624f42fe01d53e7ed1